plugins/tag_search/lib/model/index/TagIndex.php

```diff
@@ -43,7 +43,7 @@ public static function getIndexFieldsMap()
 		{
 			self::$fieldsMap = array(
 				'int_id' => 'intId',
-				'tag' => 'TagWithEqual',
+				'tag' => 'tag',
```

Comment:
This was committed by Noam Arad. @noam-arad , could you please explain this implementation?